.github/workflows/cqrs-test.yml

```diff
@@ -37,13 +37,6 @@ jobs:
       - name: Dart version
         run: dart --version
 
-      - name: Cache pub dependencies
```

Comment:
There's no flutter-action so there's no caching. You're removing the only caching step